docker-compose.yml

```diff
@@ -1,9 +1,39 @@
 version: '3'
+
+networks:
+  lms_default:
+    external: true
+    name: lms_default
+
 services:
   postgres:
     image: postgres:11.5-alpine
+    networks:
+      lms_default: { }
     ports:
       - '127.0.0.1:5433:5432'
     healthcheck:
         test: ["CMD", "pg_isready", "-U", "postgres"]
         interval: 1s
+
+  web:
+    image: hypothesis/lms:dev
+    networks:
+      lms_default: { }
+    ports:
+      - '127.0.0.1:8001:8001'
+    # Unfortunately if we don't run as root `dev_host_bridge.sh` doesn't work
+    user: root
+    command: /bin/sh -c "/var/lib/lms/bin/dev_host_bridge.sh && bin/init-env supervisord -c conf/supervisord.conf"
+    env_file: .devdata.env
+    environment:
+      - DATABASE_URL=postgresql://postgres@lms_postgres_1/postgres
+      - FEATURE_FLAGS_COOKIE_SECRET=not_a_secret
+      - H_API_URL_PRIVATE=http://host.docker.internal:5000/api/
```

Comment:
This is where we use the host we setup

Reply:
Where do all these envvars come from when the non-Docker `make dev` is run?

I can see that some of these are Docker-specific (like `H_API_URL_PRIVATE=http://host.docker.internal:5000/api/`) but many of them don't look Docker-specific and look like the same values that `make dev` might use

Reply:
Some come from `tox.ini` most come from `conf/development.ini`.

Reply:
Yeah, I don't see any reason not to move them into `tox.ini`